js/app.js

```diff
@@ -63,12 +71,21 @@ vote.photoArray.push(new Photo('img/kittens/cat14.jpg'));
 
 var lphoto = document.getElementById('leftphoto');
 var rphoto = document.getElementById('rightphoto');
+var results = document.getElementById('results');
```

Comment:
This was a good start, passable. I would have liked to see more jQuery on things like the `document.getElementById` assignments, but I don't know what else you had going on... I've seen your able to use jQuery though so just a thought.